presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -778,7 +766,7 @@ void PrestoTask::updateExecutionInfoLocked(
       prestoTaskStats);
 }
 
-/*static*/ std::string PrestoTask::taskNumbersToString(
+/*static*/ std::string PrestoTask::taskStatesToString(
```

Comment:
These are more of part of taskmanager state, not task state. We can keep the original name?

Reply:
It means the task state but not the internal execution state. Task number is confusing here.

Reply:
Keeping as is.